apps/desktop-wallet/src/api/utilHooks.ts

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Seems like this file contains hooks to get info about token types. Should it be named accordingly?

Reply:
I wrote about this file here: https://github.com/alephium/alephium-frontend/pull/748#discussion_r1690410473

Reply:
But anyhow, don't you think that the name of the file should reflect its content, rather than how it should be used? As you said in your comment, we don't know if those hooks could be useful in components in the future (they very well may be).

By calling this a "util" file I just fear that it'll become a messy file with may random hooks inside :)

Reply:
Ok! Let's start like this.